Pin Dropper/PinsListViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  PinsListViewController.swift
+//  Pin Dropper
+//
+//  Created by Talha Babar on 7/2/15.
+//  Copyright (c) 2015 Talha Babar. All rights reserved.
+//
+
+
+class PinsListViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    var passed : [Pin]!
+    var cellIdentifier = CustomCell.cellIdentifier
```

Comment:
- This can be a `let`
- You don't need to store it in a `var`/`let`... It can directly be used in the `dequeCellWith...` call